paymentsheet/src/test/java/com/stripe/android/paymentsheet/model/ConfirmPaymentIntentParamsFactoryTest.kt

```diff
@@ -76,6 +81,73 @@ class ConfirmPaymentIntentParamsFactoryTest {
         )
     }
 
+    @Test
+    fun `create() with saved card and shippingDetails sets shipping field`() {
+        val shippingDetails = AddressDetails(
+            name = "Test",
+            address = PaymentSheet.Address(
+                line1 = "line1",
+                city = "city"
+            ),
+            phoneNumber = "5555555555"
+        )
+        val factoryWithConfig = ConfirmPaymentIntentParamsFactory(
+            PaymentIntentClientSecret(CLIENT_SECRET),
+            shippingDetails.toConfirmPaymentIntentShipping()
+        )
+
+        Truth.assertThat(
+            factoryWithConfig.create(
+                paymentSelection = PaymentSelection.Saved(
+                    PaymentMethodFixtures.CARD_PAYMENT_METHOD
+                )
+            )
+        ).isEqualTo(
+            ConfirmPaymentIntentParams.createWithPaymentMethodId(
+                paymentMethodId = "pm_123456789",
+                clientSecret = CLIENT_SECRET,
+                paymentMethodOptions = PaymentMethodOptionsParams.Card(
+                    setupFutureUsage = ConfirmPaymentIntentParams.SetupFutureUsage.Blank
+                ),
+                shipping = shippingDetails.toConfirmPaymentIntentShipping()
+            )
+        )
```

Comment:
Same for this test and the one below: I think asserting only the part which we’re interested in (the shipping field) makes more sense and is more in line with the test’s name.